### ISSUE TITLE
(Accessibility)A11y_VsCode_GoToFile_ScreenReader:Nvda is not reading placeholder text.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51832]
Environment Details:
VSCode Version : 1.24.1
OS Version : Win10
Additional Details:
NVDA 2018.2,JAWS18.0.5038,MAS1.3.1
same issue is observed in JAWS
Repro Steps:
1)Launch VSCode.
2)Navigate   to Go(Alt+G) menu option and select "Go To File" from dropdown.
3)Check with NVDA reading.
Actual:
NVDA does not read the placeholder present on the screen.
Expected:
NVDA should read the placeholder which is present on the screen.
Recommendations:
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx
](url)
[User] Impact:
User will not able to know that there is placeholder present on the screen.
MAS Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}
Attachment for Reference:
![accessibility a11y_vscode_gotofile_screenreader](https://user-images.githubusercontent.com/39204909/41394911-56d1c710-6fc9-11e8-9a13-0e347014b0f8.png)
Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/51824
